pallets/automation-time/src/lib.rs

```diff
@@ -1011,7 +1011,15 @@ pub mod pallet {
 				},
 			}
 
+			// TODO: Additional read/writes accounts for AssetRegistry assets unavailable.
```

Comment:
modifying the weight here means no need to manually update github action weights manually.